subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -413,6 +413,13 @@ config BT_CTLR_SCAN_INDICATION
 	help
 	  Generate events indicating on air scanner events.
 
+config BT_CTLR_SCAN_UNRESERVED
+	bool "Unreserved scan window"
+	depends on BT_OBSERVER
+	help
+	  Scanner will not use time space reservation for scan window when in
+	  continuous scan mode.
```

Comment:
Is there any downside to this? Could it be `default y` or `default y if BT_MESH`?

Reply:
`default y if BT_MESH` should be ok. But I recommend, this option be set explicit in mesh sample to make it visible to users.

```
CONFIG_BT_CTLR_ADVANCED_FEATURES=y
CONFIG_BT_CTLR_SCAN_UNRESERVED=y
```